pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -509,7 +509,7 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 			tr("Completely skips drawing surfaces from the surface in the left box up to the surface specified in the box on the right."));
 
 		dialog->registerWidgetHelp(m_ui.hwAutoFlush, tr("Auto Flush"), tr("Unchecked"),
-			tr("Force a primitive flush when a framebuffer is also an input texture. "
+			tr("Force a primitive flush when a frame buffer is also an input texture. "
```

Comment:
Framebuffer is also correct language. Frame buffer is actually alternative but still correct.

Reply:
Framebuffer is correct https://www.khronos.org/opengl/wiki/Framebuffer

Reply:
Restoring everything.